test/TensorFlow/debug_scope.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-swift-frontend -Xllvm -tf-dump-intermediates -emit-sil %s -verify
+// RUN: %target-swift-frontend -Xllvm -tf-dump-intermediates -Xllvm -tf-strict-deabstraction=false -emit-sil %s -verify
```

Comment:
FYI: this test was removed in https://github.com/apple/swift/pull/18297, you can remove it also.

Reply:
even better! :)